src/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Checkbox as CoreCheckbox } from 'wix-ui-core/checkbox';
+import styles from './Rating.st.css';
+import { ReactComponent as StarIcon } from '../../assets/icons/Star.svg';
+
+export enum IconSize {
+  LARGE = 'large',
```

Comment:
Lets use capitalized for the enum props:
```
Large = 'large'
```
I know there is some inconsistency in the library, but I want to start enforcing this.

Reply:
And we should also add a `small` size, which will be the default size.

Reply:
Will do